contracts/token/Minter.sol

```diff
@@ -240,13 +300,20 @@ contract Minter is Manager, IMinter {
             currentBondingRate = MathUtils.percPoints(totalBonded, totalSupply);
         }
 
-        if (currentBondingRate < targetBondingRate) {
+        // Adjust inflation based on current bonding rate and target bonding rate, ensuring it stays within the floor and ceiling
+        if ((currentBondingRate < targetBondingRate && inflation < maxInflation) || inflation < minInflation) {
```

Comment:
Interesting, so we want the inflation update to be gradual, even if the governors set a min/max inflation way lower than current inflation. I thought it could happen that we'd immediately set the inflation to the given min/max on the next round, but this is probably better for tokenomics stability and requires a strategy from governors.